decoder.rb

```diff
@@ -0,0 +1,36 @@
+morse_message = '.-   -... --- -..-   ..-. ..- .-.. .-..   --- ..-. / .-. ..- -... .. . ...'
+
+def decode_char(morse_char)
+  morse_dictionary = {
+    'A' => '.-', 'B' => '-...', 'C' => '-.-.',
+    'D' => '-..', 'E' => '.', 'F' => '..-.',
+    'G' => '--.', 'H' => '....', 'I' => '..',
+    'J' => '.---', 'K' => '-.-', 'L' => '.-..',
+    'M' => '--', 'N' => '-.', 'O' => '---',
+    'P' => '.--.', 'Q' => '--.-', 'R' => '.-.',
+    'S' => '...', 'T' => '-', 'U' => '..-',
+    'V' => '...-', 'W' => '.--', 'X' => '-..-',
+    'Y' => '-.--', 'Z' => '--..'
+  }
+  morse_dictionary.key(morse_char) || ' '
+end
+
+def decode_word(morse_word)
+  word = ''
+  morse_word.split.each do |char|
+    word += decode_char(char)
+  end
+  word
+end
+
+def decode_message(morse_text)
+  message = []
+  morse_text.split('   ').each do |word|
+    message.push(decode_word(word))
+  end
+  message.join(' ')
+end
+
```

Comment:
Please add a `decode` method which uses `\#decode_word`. `p decode("-- -.-- -. .- -- .")` should return `"MY WORD" and p decode(".- -... --- -..- ..-. ..- .-.. .-.. --- ..-. .-. ..- -... .. . ...")` should return `"A BOX FULL OF RUBIES"`

Reply:
Dear @SajjadAhmad14 👍 

Thanks for the review. It is fixed.

Regards

Reply:
Nice work! This issue has been addressed well! :clap: :clap: :clap:

Reply:
👍 👍 👍